ert/tracker/evaluator_tracker.py

```diff
@@ -25,33 +18,75 @@
     FullSnapshotEvent,
     SnapshotUpdateEvent,
 )
+from ert_shared.status.entity.state import (
+    ENSEMBLE_STATE_CANCELLED,
+    ENSEMBLE_STATE_STOPPED,
+    ENSEMBLE_STATE_FAILED,
+)
+from ert_shared.status.utils import state
+
+if TYPE_CHECKING:
+    from ert3.evaluator._evaluator import ERT3RunModel
+    from ert_shared.models.base_run_model import BaseRunModel
+    from cloudevents.http.event import CloudEvent  # type: ignore
 
 
 class OutOfOrderSnapshotUpdateException(ValueError):
     pass
 
 
+class EvaluatorConnectionInfo:
```

Comment:
I was just wondering why this was moved? And also why it wasn't moved into e.g. `ert_shared/ensemble_evaluator/conn_info.py` or something?

Reply:
The main reason is that `ert_shared` does not undergo type (pylint, mypy) checking. On the other side we actually thought about moving it to a stand-alone file, but since `EvaluatorConnectionInfo` is currently mostly used in `EvaluatorTracker` we decided to have it there. I guess it is open to discussion ....